docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/configuring-tracker-plugins/javascript/index.md

```diff
@@ -103,3 +103,48 @@ index a9ec92f7..be81d785 100644
 ```
 
 In this case the resulting `sp.js` file will contain the `my_context` entity from the plugin, but the generated `sp.lite.js` will not.
+
+## Plugin Options
+
+Some plugins are configurable and take options in as parameters.
+The options available are described in each plugin's documentation.
+
+How you pass these options to the plugin varies by when the plugin is loaded:
+
+### Bundled Plugins
+
+Bundled plugins will automatically include any exposed methods.
+Additional configuration is typically passed through via the [`contexts`](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/initialization-options/index.md#adding-predefined-contexts) configuration, which is extracted and passed to the plugin in `features.ts`, described in [Including Custom Plugins](#including-custom-plugins).
+
+```javascript
+snowplow('newTracker', 'sp', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  contexts: {
+    gaCookies: { ga4: true, ua: false }
+  },
+  plugins:
+});
+```
+
+### Plugins After Initialization
```

Comment:
This is super useful!

Might be worth adding a link here from the [plugin options section in the Web vitals plugin page](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/web-vitals/#plugin-options).